### PR TITLE
Track separable states in UI

### PR DESCRIPTION
**Pull request by gwwatkin**

### CODE REVIEW

**Comment by gwwatkin** [comment]
closes #47

**Review by alexnguyenn — APPROVED** [review]

**Comment by alexnguyenn** [comment]
Note that the spelling for `DefalutSymbolicStates` are incorrect.

**Comment by gwwatkin** [comment]
Great catch there

File: webgui/templates/lattice_view.mak [hunk 1]
```diff
@@ -125,7 +125,7 @@
                                </span><br>
                                <span>
                                    %if cell is not None and cell.text is not None:
-                                        <span style="font-size: 17px"><b>${cell.text}</b></span>
+                                        <span style="font-size: 8px"><b>${cell.text}</b></span>
```
Comment (gwwatkin) [inline webgui/templates/lattice_view.mak:128]: @Keelando This probably needs to be ported to the new interface

File: webgui/lattice_view.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def sparse_lattice_to_array(lattice: patches.Lattice) -> List[List[Optional[Visu
             array[y][x] = VisualArrayCell(patch.patch_type, {})
             if(patch.state is not None):
                 if(cell_idx_in_patch == 0): # Only display the value in the first cell of the patch
-                    array[y][x].text = patch.state.ket_repr()
+                    array[y][x].text = patch.state.ket_repr().replace("\n","<br>")
```
Comment (gwwatkin) [inline webgui/lattice_view.py:50]: @Keelando This too

**Pull request merged**